### ISSUE TITLE
legacy QuantUS question

### ISSUE THREAD

**Issue by sltsang** [QuantUS-OpenSource/QuantUS#39]
---
name: AUC Value calculation? - Using Lognormal Parameter Instead of Actual Area Under Curve
about: Ask question
title: ''
labels: ''
assignees: ''
---

## Issue Type
- [ ] Bug Report
- [ ] Feature Request
- [ ] Installation Issue
- [x] Question/Help
- [ ] Documentation
- [ ] Other

## Description
The AUC (Area Under Curve) value displayed in the 2D DCE CEUS analysis is not the actual integrated area under the TIC curve. Instead, it's reporting the auc scaling parameter from the lognormal curve fit, which is a different quantity.

There's a commented line at line 47 suggesting this was intentional:` print("brute force auc:", sklearn.metrics.auc(TIC[0], wholeCurve))` and I'm wondering why this method is used instead.

The issue I'm having is that:
  - Unusually high AUC values even for necrotic/low-signal regions
  - AUC values that don't correlate with visual assessment of the TIC

## Environment (for bugs/installation issues)
- OS: 
- Python Version: 
- Package Version: 
- Installation Method: 

## Steps to Reproduce (for bugs)
1. 
2. 
3. 

## Expected vs Actual Behavior (for bugs)
**Expected:** 
  Option 1: Integrate the fitted curve
 ` auc = np.trapz(wholeCurve, x=TIC[0])`

  Option 2: AUC scaled back to original intensity values
 ` auc = np.trapz(wholeCurve * normalizer, x=TIC[0])`

  Or use the wholeCurve after it's scaled (line 53):
  wholeCurve *= normalizer
  `auc = np.trapz(wholeCurve, x=TIC[0])`
**Actual:** 
  ```
def data_fit(TIC, normalizer, autoT0):
      # ... curve fitting code ...

      popt = normalizedLogParams

      auc = popt[0]  # <-- This is the lognormal scaling parameter, NOT the true AUC
      mu = popt[1]
      sigma = popt[2]
      mtt = np.exp(mu + (sigma**2 / 2))
      tp = np.exp(mu - (sigma**2))
      pe = np.max(wholeCurve)
      params = np.array(np.around(np.array([pe, auc, tp, mtt, t0]), decimals=1))
```

  The lognormal function is defined as:
```
  def lognormal(x, auc, mu, sigma):
      curve_fit = (auc / (2.5066 * sigma * x)) * np.exp(
          (-1 / 2) * (((np.log(x) - mu) / sigma) ** 2)
      )
      return np.nan_to_num(curve_fit)

```
## Error Messages/Logs (if applicable)
```
Paste error messages here
```

## Additional Context
Location:
src/Utils/lognormalFunctions.py, lines 43-50

**Comment by HybridEcho** [comment]
so we can close this issue?
